cmd/githubgen/go.sum

```diff
@@ -1,73 +0,0 @@
-github.com/ProtonMail/go-crypto v0.0.0-20230217124315-7d5c6f04bbb8 h1:wPbRQzjjwFc0ih8puEVAOFGELsn1zoIIYdxvML7mDxA=
```

Comment:
This deleted file seems not related to this pr and cause the failed lint, could you revert this file?

Reply:
Reverted. Looks like this came in with mod tidy.